site/sfguides/src/building-snowflake-llm-based-functions-using-coalesce/building-snowflake-llm-based-functions-using-coalesce-and-iceberg.md

```diff
@@ -0,0 +1,587 @@
+author: Josh Hall
+id: building-snowflake-llm-based-functions-using-coalesce-and-iceberg
+summary: Learn to build Cortex based data pipelines in Coalesce using Iceberg tables
+categories: data-engineering
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Data Engineering, Getting Started
+
+# Building Snowflake LLM Based Functions using Coalesce and Iceberg
+<!-- ------------------------ -->
+## Overview 
+Duration: 1
+
+This Hands-On Lab exercise is designed to help you learn how to build and manage Iceberg tables with Snowflake Cortex LLM nodes within Coalesce. In this lab, you’ll explore the Coalesce interface, load Iceberg tables into your project, learn how to easily transform and model your data with our core capabilities, and use the Cortex LLM functions node that Coalesce provides users. 
+
+### Prerequisites
+- A basic understanding of SQL 
+
+### What You’ll Learn 
+- How to navigate the Coalesce interface
+- Load in iceberg tables
+- How to add data sources to your graph 
+- How to prepare your data for transformations with Stage nodes
+- How to union tables
+- How to set up and configure Cortex LLM Nodes
+- How to analyze the output of your results in Snowflake
+
+
+### What You’ll Need 
+- FA Snowflake trial account
```

Comment:
@joshdhall Pl add link to the Snowflake signin/signup page.

Reply:
@iamontheinet this is done. Thanks!